motion_tracker/utils/image_cropping.py

```diff
@@ -187,9 +187,13 @@ def crop_and_resize(img, img_coords, box_coords, output_width, output_height, ra
                  If False, create crop based on true bounding box
                  Cropped area is twice as large (and centered) on box either way
     '''
-    crop_coords, box_after_crop = get_crop_coords(box_coords, img_coords, random_crop)
-    cropped_img = img[crop_coords.y0:crop_coords.y1,
-                      crop_coords.x0:crop_coords.x1]
+
+    # Placeholder to get into the while loop the first time. 
+    cropped_img = np.zeros((2, 2))
+    while 0 not in cropped_img.shape: 
+        crop_coords, box_after_crop = get_crop_coords(box_coords, img_coords, random_crop)
+        cropped_img = img[crop_coords.y0:crop_coords.y1,
+                          crop_coords.x0:crop_coords.x1]
```

Comment:
@dansbecker This was the major change I made. It turns out that it was strictly the center of the randomly cropped bounding box that caused our issue. That happened to always be true when we ran into the resizing error, but if I simply did another random crop every time that was true, it was too frequent. I tried adding another couple of stipulations (the cropped area x0 and y0 had to be less than the bounding box x0 and y0), but in the end I think it's ultimately a kind of conglomerate of things that ends up causing the issue. 

Probably what matters most is that I'm fairly certain that it's not a bug in our code or something that's written incorrectly, but rather just a conglomerate of the laplace parameters just the right image size and bounding box combo. This ultimately manifests itself here when we try to actually grabbed the cropped image and then resize that cropped image - when we get the error we have negative indices here, and so the cropped image ends up having a 0 dimension along one of it's axis (hence the use of the `while 0 not in cropped_img.shape`. If you're good with doing it this way, I'll clean up this part, and then do any refactoring in this file that I can see (I don't think there's much, but we discussed at one point doing some refactoring to this file).